src/sage/rings/multi_power_series_ring_element.py

```diff
@@ -652,12 +652,20 @@ def _im_gens_(self, codomain, im_gens, base_map=None):
                     b |--> 2*y
             sage: phi(a+b+3*a*b^2 + A.O(5))  # indirect doctest
             x + 2*y + 12*x*y^2 + O(x, y)^5
+
+        Check that :issue:`37729` has been fixed::
+
+            sage: A.<a,b> = PowerSeriesRing(ZZ)
+            sage: A.hom([0,0],ZZ)(1 - a^3 + b^5)
```

Comment:
```suggestion
            sage: A.<a,b> = PowerSeriesRing(ZZ)
            sage: phi = A.hom([0,0],ZZ)
            sage: phi(1 - a^3 + b^5)
```
Let's spread it out to make it a little easier to read.